packages/@aws-cdk/aws-ec2/lib/instance-types.ts

```diff
@@ -556,7 +571,13 @@ export class InstanceType {
     return new InstanceType(`${instanceClass}.${instanceSize}`);
   }
 
+  /**
+   * The instance's CPU architecture
+   */
+  public readonly architecture: InstanceArchitecture;
```

Comment:
Wasn't sure if it would have been better to forgo a new property and instead just make the `resolveArchitecture` method public. Thoughts?

Reply:
I like the current public name. The best of both worlds would be to make `resolveArchitecture` a getter :)

```ts
public get architecture(): InstanceArchitecture {
  // ...body of resolveArchitecture here...
}
```